docs/development/building-and-testing-packages.md

```diff
@@ -142,27 +144,38 @@ and you can pass options to it just like normal. Currently `subprocess` doesn't
 work, so if you have a test runner that uses `subprocess` then it cannot be
 used.
 
-## Build Github actions example
+## Build Github actions examples
 
-Here is a complete example of a Github Actions workflow for building a Python
-wheel out of tree:
+This is a complete example for building a Python wheel out of tree using
+`pyodide build` directly:
 
 ```yaml
 runs-on: ubuntu-22.04
```

Comment:
```suggestion
runs-on: ubuntu-22.04 # or ubuntu-latest
```

I think we should specify that the `ubuntu-22.04` image is not a hard requirement